io.c

```diff
@@ -13331,6 +13331,163 @@ set_LAST_READ_LINE(VALUE val, ID _x, VALUE *_y)
+ *
+ *  === Positioning
+ *
+ *  - #lineno:: Returns the current line number in +self+.
+ *  - #lineno=:: Sets the line number is +self+.
+ *  - #pos:: Returns the current offset (bytes) in +self+.
+ *  - #pos=:: Sets the offset (bytes) in +self+.
+ *  - #reopen:: Reassociates +self+ with a new or existing I/O stream.
+ *  - #rewind:: Positions +self+ to the beginning of input.
+ *  - #seek:: Sets the offset for +self+.
+ *  - #tell:: Returns the current offset (bytes).
+ *
+ *  === Iterating
+ *
+ *  - ::foreach:: Calls the given block with each successive file line
+ *                for the given filepath.
+ *  - #each (aliased as #each_line):: Calls the given block
+ *                                    with each successive line in +self+.
+ *  - #each_byte:: Calls the given block with each successive byte in +self+.
+ *  - #each_char:: Calls the given block with each successive character in +self+.
+ *  - #each_codepoint:: Calls the given block with each successive codepoint in +self+.
+ *  - #each_line:: Calls the given block with each successive codepoint in +self+.
+ *
+ *  === Converting
+ *
+ *  - ::try_convert:: Returns the new \IO object resulting from converting
+ *                    the given object.
+ *  - #inspect:: Returns the string representation of +self+.
+ *
+ *  === Settings
+ *
+ *  - #autoclose=:: Sets whether +self+ auto-closes.
+ *  - #binmode:: Sets +self+ to binary mode.
+ *  - #close_on_exec=:: Sets the close-on-exec flag.
+ *  - #sync=:: Sets the sync-mode to the given value.
+ *
+ *  === Querying
+ *
+ *  - #autoclose?:: Returns whether +self+ auto-closes.
+ *  - #binmode?:: Returns whether +self+ is in binary mode.
+ *  - #close_on_exec?:: Returns the close-on-exec flag for +self+.
+ *  - #closed?:: Returns whether +self+ is closed.
+ *  - #eof? (aliased as #eof):: Returns whether +self+ is at end-of-file.
+ *  - #fileno (aliased as #to_i):: Returns the integer file descriptor for +self+
+ *  - #sync:: Returns whether +self+ is in sync-mode.
+ *  - #tty (aliased as #isatty):: Returns whether +self+ is a terminal.
+ *
+ *  === Closing
+ *
+ *  - #close:: Closes +self+.
+ *  - #close_read:: Closes +self+ for reading.
+ *  - #close_write:: Closes +self+ for writing.
+ *
+ *  === Encoding
+ *
+ *  - #external_encoding:: Returns the external encoding object for +self+.
+ *  - #internal_encoding:: Returns the internal encoding object for +self+.
+ *  - #set_encoding:: Sets the encoding for +self+.
+ *  - #set_encoding_by_bom:: Sets the encoding for +self+, based on its
+ *                           Unicode byte-order-mark.
+ *
+ *  === Buffering
+ *
+ *  - #fdatasync:: Immediately writes all buffered data in +self+ to disk.
+ *  - #flush:: Flushes any buffered data within +self+ to the underlying
+ *             operating system.
+ *  - #fsync:: Immediately writes all buffered data in +self+ to disk.
+ *  - #ungetbyte:: Pushes the given byte or string onto +self+.
```

Comment:
`Prepends buffer for +self+ with given integer byte or string`.

Reply:
Fixed.

Reply:
This change doesn't appear to have been made.